packages/cli/src/commands/doctor/healthchecks/__tests__/androidSDK.test.ts

```diff
@@ -0,0 +1,66 @@
+import execa from 'execa';
+import androidSDK from '../androidSDK';
+import getEnvironmentInfo from '../../../../tools/envinfo';
+import {EnvironmentInfo} from '../../types';
+import {NoopLoader} from '../../../../tools/loader';
+
+import * as common from '../common';
+
+const logSpy = jest.spyOn(common, 'logManualInstallation');
+
+const mockExeca = (stdout: string) => {
```

Comment:
Let's rename it to `mockSdkManagerResponse` or something so that it's closer to what it's actually doing, since we don't extract execa call to a function we could mock – which would make it obvious what is mocked.

Reply:
See what I changed. What do you think?